app/models/comment.rb

```diff
@@ -0,0 +1,5 @@
+class Comment < ApplicationRecord
```

Comment:
- [ ] it should Comment model has a method that updates the comments counter for a post. please I encourage you to do that